buildspecs/linux_arm_linaro.spec

```diff
@@ -227,8 +227,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<flag id="opt_jvmti" value="true"/>
 		<flag id="opt_jxeLoadSupport" value="true"/>
 		<flag id="opt_memoryCheckSupport" value="true"/>
-		<flag id="opt_multiVm" value="true"/>
-		<flag id="opt_panama" value="false"/>
+		<flag id="opt_multiVm" value="true"/>s
```

Comment:
The trailing `s` does not belong. Please remove.

Reply:
Must have come while saving the file, missed it while staging commits as well, my bad. Fixed it!